source/tests/system/nirfmxbt_driver_api_tests.cpp

```diff
@@ -258,14 +244,34 @@ TEST_F(NiRFmxBTDriverApiTests, TxpBasicFromExample_DataLooksReasonable)
   EXPECT_GT(fetched_powers_response.peak_to_average_power_ratio_maximum(), 0.0);
 }
 
-// TODO -- AB#1835516: Add system-level test using an API function that uses complex data types.
-TEST_F(NiRFmxBTDriverApiTests, SetAttributeComplex_ExpectedError)
+TEST_F(NiRFmxBTDriverApiTests, ExerciseComplexNumberUsageWithModAccFetchConstellationTrace_DataLooksReasonable)
```

Comment:
We use _ to denote the 3 phases of a specific test that correlate with the newlines in the test code. The 3 phases being "Given_When_Then" a different name could be "ModAccMeasurement_FetchConstellationTrace_ComplexNumberLooksReasonable"

Reply:
Good catch. I've changed the test name to your recommendation.